app/Http/Controllers/Api/ClientController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Resources\ClientResource;
+use App\Models\Client;
+use Symfony\Component\HttpFoundation\Response;
+use Illuminate\Http\ClientStoreRequest;
+
+class ClientController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+
+        return ClientResource::collection(Client::with('projects')->get());
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\ClientStoreRequest  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(\App\Http\Requests\ClientStoreRequest $request)
```

Comment:
@faxunil why don't you add `use App\Http\Requests\ClientStoreRequest` on top instead, and then you wouldn't need to use full path here